devlib/collector/ftrace.py

```diff
@@ -61,7 +61,6 @@ def __init__(self, target,
                  trace_children_functions=False,
                  buffer_size=None,
                  top_buffer_size=None,
-                 buffer_size_step=1000,
```

Comment:
This can't be removed without a warning and a deprecation period as this is part of the public API.

Reply:
OK. Dropped that much for now.

Reply:
Yeah some cleanup is overdue, I think this is sort of cleanup is on the roadmap of the next maintainers